sdk/resourcemanager/azure-resourcemanager-network/src/test/java/com/azure/resourcemanager/network/ApplicationGatewayTests.java

```diff
@@ -38,6 +38,9 @@
 
     @Test
     public void canCRUDApplicationGatewayWithWAF() throws Exception {
+        if (skipInPlayback()) {
+            return; // removed pfx
```

Comment:
We should be able to create app gateway without pfx (use http instead of https). Later task.